### DISCUSSION TITLE
"Permission denied by system" for chrome launched from wxt

### DISCUSSION THREAD

**Discussion by ming535** [wxt-dev/wxt#667] [General]
I was getting error of "Permission denied by system" when using `navigator.mediaDevices.getUserMedia` when I use the chrome browser launched by `wxt`. The error does not happen when I use my normal Chrome.

For example, this extension works fine on my own Chrome https://github.com/hacess/chrome-extension-manifest-v3-window-recording

But when I was using wxt and load the extension using the browser launched from wxt, it fails on permission error.
<img width="1223" alt="Screenshot 2024-05-22 at 21 45 42" src="https://github.com/wxt-dev/wxt/assets/7576663/e1343a91-abb6-492c-a35f-d3ba01bd18c7">

When I load this extension using my normal Chrome browser, there is no error.

I raise the discussion using this extension, and load the extension using the browser from wxt because I was building my own extension  using the api `navigator.mediaDevices.getUserMedia` and got the same error.

I suppose there is some weird thing happened when Chrome is launched from wxt?

Also, is there a way to just use my local Chrome browser, not the one started by wxt (I understand that they both use the same Chrome but using different command line argument)?

**Comment by ming535** [comment]
The same error can also be reproduced by disable the "Privacy & Security --> Screen Record" for normal Chrome.
It seems that MacOS doesn't apply the "Privacy & Security --> Screen Record" to the Chrome started by wxt.
Is there a way to lunch the Chrome browser just use my normal Chrome browser when using wxt?

**Comment by ming535** [comment]
I am able to get ride of the exception without using `web-ext`.
```typescript
import { defineRunnerConfig } from "wxt";
export default defineRunnerConfig({
  startUrls: ["https://www.youtube.com"],
  disabled: true,
});
```
But hot reload is not working any more. Is there a way to make hot reload working again?

**Reply by aklinker1** [reply]
Reloading should work fine if you disable the runner and manually install it... Are there any error logs in the background's console?
